src/Microsoft.Diagnostics.DebugServices.Implementation/ModuleService.cs

```diff
@@ -134,14 +136,17 @@ IModule IModuleService.GetModuleFromAddress(ulong address)
                 Debug.Assert((start & ~RawMemoryService.SignExtensionMask()) == 0);
                 ulong end = start + module.ImageSize;
 
-                if (address >= start && address < end) {
+                if (address >= start && address < end)
```

Comment:
I wish we did enforce this for one line blocks :)

Reply:
Maintained this behavior.